ingress/controllers/gce/controller/utils.go

```diff
@@ -34,7 +34,10 @@ import (
 	"github.com/golang/glog"
 )
 
-const allowHTTPKey = "kubernetes.io/ingress.allowHTTP"
+const (
+	allowHTTPKey    = "kubernetes.io/ingress.allow-http"
```

Comment:
and if it is GCE specific, we should probably call it `gce.ingress.kubernetes.io` or something